tests/test_keyboard_shortcuts.py

```diff
@@ -0,0 +1,279 @@
+    page.job_details.wait_for_region_to_load()
+
+    page.select_next_job()
+    page.job_details.wait_for_region_to_load()
+
+    assert page.job_details.job_keyword_name == assumed_job_keyword
+
+
+def test_previous_job_shortcut(base_url, selenium):
+    """Shortcut: 'Left Arrow'
+    Open Treeherder page, select random job, select previous job, take
+    job keyword, go back to next job and select previous job using Left Arrow
+    shortcut, verify if job keywords match"""
+    page = TreeherderPage(selenium, base_url).open()
+    all_jobs = page.all_jobs
+
+    # Check number of jobs
+    num_of_jobs = len(all_jobs) - 1
+    rnd_number = random.randint(0, num_of_jobs)
+    previous_job = rnd_number - 1
+
+    if rnd_number == 0:
+        previous_job = num_of_jobs
+
+    # Select random job and job to the left
+    all_jobs[rnd_number].click()
+    all_jobs[previous_job].click()
+    page.job_details.wait_for_region_to_load()
+    assumed_job_keyword = page.job_details.job_keyword_name
+
+    all_jobs[rnd_number].click()
+    page.job_details.wait_for_region_to_load()
+
+    page.select_previous_job()
+    page.job_details.wait_for_region_to_load()
+
+    assert page.job_details.job_keyword_name == assumed_job_keyword
+
+
+def test_previous_unclassified_failure_shortcut(base_url, selenium):
+    """Shortcut: 'p'
+    Open Treeherder page, show only unclassified failures, select random
+    failure, select previous failure, take job keyword, go back to next failure
+    and select previous unclassified failure using 'p' button shortcut,
+    verify if job keywords match"""
+    page = TreeherderPage(selenium, base_url).open()
+    page.show_only_unclassified_failures()
+    all_unclassified_failures = page.all_jobs
+
+    # Check number of unclassified failures
+    num_of_unclass_failures = len(all_unclassified_failures) - 1
+    rnd_number = random.randint(0, num_of_unclass_failures)
+    previous_failure = rnd_number - 1
+
+    if rnd_number == 0:
+        previous_failure = num_of_unclass_failures
+
+    # Select random unclassified failure
+    all_unclassified_failures[rnd_number].click()
+    all_unclassified_failures[previous_failure].click()
+    assumed_job_keyword = page.job_details.job_keyword_name
+
+    all_unclassified_failures[rnd_number].click()
+    page.job_details.wait_for_region_to_load()
+
+    page.select_previous_unclassified_failure()
+    page.job_details.wait_for_region_to_load()
+
+    assert page.job_details.job_keyword_name == assumed_job_keyword
+
+
+def test_select_next_info_tab_shortcut(base_url, selenium):
+    """Shortcut: 't'
+    Open Treeherder page, select random job, get active tab name, depending on
+    tab header name verify if next tab have assumed value, change tabs using
+    't' keyboard shortcut"""
+    page = TreeherderPage(selenium, base_url).open()
+    page.select_random_job()
+
+    active_tab_name = page.job_details.active_tab_name
+
+    if active_tab_name == 'Job details':
```

Comment:
This test should be deterministic. We should expect the first tab to be job details. Otherwise all the if blocks will be skipped and the test would pass. You could make the tab names a list, and simply loop through them.